Sem. 09/Task2.cpp

```diff
@@ -0,0 +1,43 @@
+//Задача 2 Напишете функции toUpper и toLower, които приемат стринг и променят всички главни букви в малки/всички малки букви в главни.
+#include <iostream>
+constexpr int DIFFERENCE = ('a' - 'A');
+
+void printArr(const char *arr) {
+  if(!arr) return;
+  std::cout << '\n';
+  while (*arr != '\0')
+  {
+    std::cout << *arr;
+    arr++;
+  }
+  std::cout << '\n';
+}
+
+void toUpper(char *str) {
+  if(!str) return;
+  while(*str != '\0') {
+    if(*str >= 'a' && *str <= 'z') {
```

Comment:
On lines 19 and 28, instead of checking for lowercase or uppercase this way, it is better to create functions `bool isLower(char ch)` and `bool isUpper(char ch)`